### PR TITLE
Some layout improvements

### PR DESCRIPTION
**Pull request by thiagoloschi**
Adds minor layout improvements on the sidebar, topbar and the count

### CODE REVIEW

File: examples/twittermap/web/public/javascripts/timeseries/controllers.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
     countDiv.title = "Display the count information of Tweets";
     countDiv.innerHTML = [
       "<div ng-if='queried'><p id='count'>{{ currentTweetCount | number:0 }}<span id='count-text'>&nbsp;&nbsp;of&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p></div>",
-      "<p id='count'>{{ totalCount | number:0 }}<span id='count-text'>&nbsp;&nbsp;{{sumText}}</span></p>",
+      "<p id='count'>{{ totalCount | number:0 }}<span id='count-text'>{{sumText}}</span></p>",
```
Comment (MoniMoledo) [inline examples/twittermap/web/public/javascripts/timeseries/controllers.js:40]: this way "mentions" gets too close to the total count. Right ?

File: examples/twittermap/web/public/javascripts/timeseries/controllers.js [hunk 2]
```diff
@@ -95,6 +95,7 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
         controller: 'TimeSeriesCtrl',
         link: function ($scope, $element, $attrs) {
           var chart = d3.select($element[0]);
+          console.log(d3);
```
Comment (MoniMoledo) [inline examples/twittermap/web/public/javascripts/timeseries/controllers.js:98]: Remove, if not using anymore

**Review by MoniMoledo — APPROVED** [review]
I just left some minor comments, but it's good to go :+1:

File: examples/twittermap/web/public/javascripts/timeseries/controllers.js (hunk 1 above)
Comment (thiagoloschi) [reply examples/twittermap/web/public/javascripts/timeseries/controllers.js:40]: These special characters are not a proper way of spacing elements as you can't control them by css. The idea here, anyways, is to have the count lying above the span, so the horizontal space between them won't really matter anymore.

File: examples/twittermap/web/public/javascripts/timeseries/controllers.js (hunk 2 above)
Comment (thiagoloschi) [reply examples/twittermap/web/public/javascripts/timeseries/controllers.js:98]: Sorry, I might have left it when I was debugging the time-series issue.

File: examples/twittermap/web/public/javascripts/timeseries/controllers.js (hunk 1 above)
Comment (MoniMoledo) [reply examples/twittermap/web/public/javascripts/timeseries/controllers.js:40]: ok then

**Pull request merged**